backend/src/main/java/quarano/actions/DescriptionCode.java

```diff
@@ -23,7 +23,9 @@
 
 	INITIAL_CALL_OPEN_INDEX(true),
 
-	INITIAL_CALL_OPEN_CONTACT(true);
+	INITIAL_CALL_OPEN_CONTACT(true),
+
+	CONSPICUOUS_SYMPTOM(4.0f, true);
```

Comment:
Can we switch to "suspicious" as the term everywhere in the codebase? Conspicious is pretty uncommon.

Reply:
done